Formula/r/rust.rb

```diff
@@ -79,6 +78,10 @@ def llvm
   end
 
   def install
+    # Update to LLVM 19 is in the 1.83 milestone.
+    # https://github.com/rust-lang/rust/pull/131293
+    odie "`llvm@18` should be switched to `llvm`" if build.bottle? && version >= "1.83.0"
```

Comment:
Is the changelog wrong then? https://github.com/rust-lang/rust/releases/tag/1.82.0

Reply:
Nope, I was looking at the wrong update PR 😅